pysat/tests/test_meta.py

```diff
@@ -69,6 +69,20 @@ def test_inst_data_assign_meta(self):
         assert self.testInst.meta['help', 'desc'] == ''
         assert self.testInst.meta['help', 'scale'] == 'linear'
 
+    def test_inst_data_assign_meta_empty_list(self):
+        self.testInst.load(2009, 1)
+        self.testInst['help'] = {'data': self.testInst['mlt'],
+                                 'units': [],
+                                 'long_name': 'The Doors'}
+        assert self.testInst.meta['help', 'units'] == ''
+
+    def test_inst_data_assign_meta_string_list(self):
+        self.testInst.load(2009, 1)
+        self.testInst['help'] = {'data': self.testInst['mlt'],
+                                 'units': ['A', 'B'],
```

Comment:
Do we want to support assignment of inconsistent length metadata?
Doesn't have to be addressed in this pull but seeing the test raised the question.
I'm thinking we create an issue if the answer is no.

Reply:
We should discuss this at the next meeting.  Not sure what the pros and cons are